builds/jenkins-build-pack-lambda/Jenkinsfile

```diff
@@ -692,7 +706,42 @@ def send_status_email(config, build_status, email_content) {
 	}
 }
 
+def checkS3AndUpdateServerless(s3BucketName) {
+	try {
+		sh "aws s3api head-bucket --bucket $s3BucketName --output json"
+		echo "Bucket exists and have access"
+		def sedCommand = "s/events/eventsDisabled/g";
+		sedCommand = sedCommand + "; /#Start:isS3EventEnabled/,/#End:isS3EventEnabled/d"
+		sh "sed -i -- '$sedCommand' ./serverless.yml"
+		echoServerlessFile()
+		return true
+	} catch(ex) {//bucket exists but with no access
+		def res
+		try {
+			res = sh (script: "aws s3api head-bucket --bucket $s3BucketName --output json 2<&1 | grep -c 'Forbidden'", returnStdout: true).trim()
+		} catch(e) {
+			echo "Bucket does not exist"
+			return false;
+		}
+		if(res) {
+			echo "Bucket exists and don't have access"
+			error ex.getMessage()
+		}
+	}
+}
 
+def updateLambdaPermissionAndNotification(lambdaARN, s3BucketName, action) {
+	try {
+		echo "update lambda config using cli"
+		UUID uuid = UUID.randomUUID();
+		def statementId = uuid.toString();
+		sh "aws lambda add-permission --function-name $lambdaARN --statement-id $statementId --action lambda:InvokeFunction --principal s3.amazonaws.com --source-arn arn:aws:s3:::$s3BucketName --output json"
+		sh "aws s3api put-bucket-notification-configuration --bucket $s3BucketName --notification-configuration '{\"LambdaFunctionConfigurations\": [{\"LambdaFunctionArn\": \"$lambdaARN\", \"Events\": [\"$action\"]}]}' --output json"
```

Comment:
i guess you are still working on this - don't forget the get-bucket-notification-configuration above 739

Reply:
done